Assets/cqui_settingselement.xml

```diff
@@ -66,86 +66,95 @@
+                                <Slider Style="SliderControl" Anchor="C,B" Size="300,13" SpaceForScroll="0" ID="WorkIconSizeSlider" Steps="10" />
                                 <Stack Anchor="C,T" StackGrowth="Right" Padding="5">
-                                    <Label Anchor="L,C" Style="ShellOptionText" String="LOC_CQUI_CITYVIEW_SMARTWORKICONALPHA"/>
-                                    <Label String="0" Style="ShellOptionText" Anchor="L,C" ID="SmartWorkIconAlphaText" Offset="3,0" />
+                                    <Label Anchor="L,C" Style="ShellOptionText" String="LOC_CQUI_CITYVIEW_WORKICONALPHA"/>
+                                    <Label String="0" Style="ShellOptionText" Anchor="L,C" ID="WorkIconAlphaText" Offset="3,0" />
+                                </Stack>
+                                <Slider Style="SliderControl" Anchor="C,B" Size="300,13" SpaceForScroll="0" ID="WorkIconAlphaSlider" Steps="101" />
+                                <Stack Anchor="R,T" StackGrowth="Left" Padding="5">
+                                    <GridButton ID="SmartWorkIconCheckbox" Anchor="L,C" Offset="0,0" Size="300,24" Style="CheckBoxControl" String="LOC_CQUI_CITYVIEW_SMARTWORKICON" />
+                                </Stack>
+                                <Stack ID="SmartWorkIconSettings" Anchor="R,T" StackGrowth="Down">
+                                    <Line Start="0,0" End="300,0" Width="2" Color="84,77,66,255" />
+                                    <Stack Anchor="C,T" StackGrowth="Right" Padding="5">
+                                        <Label Anchor="L,C" Style="ShellOptionText" String="LOC_CQUI_CITYVIEW_SMARTWORKICONSIZE"/>
+                                        <Label String="0" Style="ShellOptionText" Anchor="L,C" ID="SmartWorkIconSizeText" Offset="3,0" />
+                                    </Stack>
+                                    <Slider Style="SliderControl" Anchor="C,B" Size="300,13" SpaceForScroll="0" ID="SmartWorkIconSizeSlider" Steps="10" />
+                                    <Stack Anchor="C,T" StackGrowth="Right" Padding="5">
+                                        <Label Anchor="L,C" Style="ShellOptionText" String="LOC_CQUI_CITYVIEW_SMARTWORKICONALPHA"/>
+                                        <Label String="0" Style="ShellOptionText" Anchor="L,C" ID="SmartWorkIconAlphaText" Offset="3,0" />
+                                    </Stack>
+                                    <Slider Style="SliderControl" Anchor="C,B" Size="300,13" SpaceForScroll="0" ID="SmartWorkIconAlphaSlider" Steps="101" />
                                 </Stack>
-                                <Slider Style="SliderControl" Anchor="C,B" Size="300,13" SpaceForScroll="0" ID="SmartWorkIconAlphaSlider" Steps="101" />
                             </Stack>
-                        </Stack>
+                        </ScrollPanel>
```

Comment:
Line 157 was added, nothing else in this area changed
![image](https://user-images.githubusercontent.com/8787640/86739429-b8afc180-bfea-11ea-8e06-e5c0f9a1f4dd.png)